frontend/jupyter/src/app/pages/form/form-default/volume/mount/mount.component.ts

```diff
@@ -1,15 +1,36 @@
-import { Component, Input, OnInit } from '@angular/core';
-import { FormGroup } from '@angular/forms';
+import { Component, Input, OnChanges, OnDestroy, OnInit, SimpleChanges } from '@angular/core';
+import { FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-volume-mount',
   templateUrl: './mount.component.html',
   styleUrls: ['./mount.component.scss'],
 })
-export class VolumeMountComponent implements OnInit {
+export class VolumeMountComponent implements OnInit, OnChanges {
   @Input() volGroup: FormGroup;
 
   constructor() {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.initComponent();
+  }
+  ngOnChanges(): void {
+    this.initComponent();
+  }
+
+  private initComponent(): void{
+    // Get the list of mounted volumes of the existing Notebooks in the selected Namespace, AAW
+
+    this.volGroup.get('mount').setValidators([
+      Validators.required,
```

Comment:
The red error box shows up for this required error, but theres no error message getting displayed. Probably need to add a mountName.hasError('required') condition to your message display function